src/components/mes-signature-pad/mes-signature-pad.scss

```diff
@@ -0,0 +1,90 @@
+mes-signature-pad {
+  .not-valid-header {
+    height: 40px;
+    .not-valid-text {
+      color: #b12025;
```

Comment:
Can you change to use the colors array using map-get

Theres a few color codes below which could also have the same done

Reply:
Yeah, there's already a colour variable for this, `$colors.mes-validation-error`.

Reply:
changed